pages/SchedulePage.tsx

```diff
@@ -28,13 +27,12 @@ interface SchedulePageProps {
 
 export default function SchedulePage({ currentShow }: SchedulePageProps) { 
 
+  const navigation = useNavigation<any>();
```

Comment:
if the type constraint here is for route name - this would be a good use of the pattern I mentioned here:

https://github.com/WMBR-MIT/wmbr-app/pull/47#discussion_r2441384599

if we export a `WmbrRouteName` we can toss it here/similar and get a little bit of type safety when we use navigation.push/replace. it's not crucial but nice - if that's the right param, then emptying hitting ctrl-space for the first param of `navigation.push(` would auto-complete the valid options